src/api.rs

```diff
@@ -34,12 +45,14 @@ pub fn register_player(
     // Broadcast to all hosts that a new player has joined.
     broadcaster.send(HostBroadcast::PlayerRegistered(PlayerData {
         id: player_id,
+        username: name_clone.clone(),
         score: 0,
     }));
 
     // Respond to the client.
     JSON(RegisterPlayerResponse {
         id: player_id,
+        username: name_clone,
```

Comment:
This (and the previous) makes sense, too. In general, when freely cloning strings that go to multiple places like this, you would want to send the original at the end, correct? That seems to make the most sense given the movement rules.